### PR TITLE
Add small tweaks to the README.

### PR DESCRIPTION
**Pull request by Yasumoto**
Small typos and added another example. This package looks so great, thank you all for the hard work!

Happy to make other tweaks or improvements y'all notice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,11 @@ To depend on the logging API package, you need to declare your dependency in you
 .package(url: "https://github.com/apple/swift-log.git", .branch("master")),
 ```
 
-and to your application/library target, add `"Logging"` to your `dependencies`.
+and to your application/library target, add `"Logging"` to your `dependencies`:
+
+```swift
+.target(name: "BestExampleApp", dependencies: ["Logging']),
+```
```
Comment (ktoso) [inline README.md]: good idea to add this, perhaps the above should lead into this though:

> and to your application/library target, add `"Logging"` to your `dependencies`, e.g. like this:
> ...

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ What `swift-log` provides today can be found in the [API docs][api-docs]. At thi
 
 ---
 
-**NOTE**: You will need Xcode 10.2 beta 4 or a recent [Swift 5.0 development snapshot](http://swift.org/download/#swift-50-development) to try out `swift-log`.
+**NOTE**: You will need Xcode 10.2 beta 4, [Xcode 10.2](https://itunes.apple.com/us/app/xcode/id497799835) or a recent [Swift 5.0 development snapshot](http://swift.org/download/#swift-50-development) to try out `swift-log`.
```
Comment (ktoso) [inline README.md]: his this a good link, won't change?

Comment (weissi) [inline README.md]: oh thanks. If you don't mind you could actually just say that you need Xcode 10.2 or Swift 5.0 . Don't think the dev snapshots or betas are still relevant and we won't unit test them anyway so I think they can be removed

Comment (Yasumoto) [reply README.md]: Looks like it to me:

> These URLs are great for linking to your app or company page in the App Store or Mac App Store from your website or inside your app, or anywhere you don't need to type or speak the URL. As a rule, always use these standard links when driving users to your app from your digital marketing communications.


https://developer.apple.com/library/archive/qa/qa1633/_index.html

Comment (Yasumoto) [reply README.md]: Ah, yah it's also linked from https://swift.org/download/#releases for the Xcode link.

Comment (Yasumoto) [reply README.md]: Great call, fixed.

**Pull request merged**